source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,190 @@
+   
+Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+
+::
+
+   >> c = parcluster;
+   
+``c = parcluster`` will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
+the correct profile will be chosen when using this command.
+
+If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+
+::
+
+   >> c = parcluster('local');
+
+You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
+other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+
+::
+
+   >> % View the additional properties
+   >> c.AdditionalProperties
+   >> % set the required properties
+   >> c.AdditionalProperties.AccountName = <account-name>;
+   >> c.AdditionalProperties.WallTime = '05:00:00';
+   >> % save the profile
+   >> c.saveProfile
+   
+Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+
+Submitting jobs
++++++++++++++++
+
+Submitting jobs will happen from within Matlab. There are two ways to do this. You can start a Matlab session on a login node, or you can first request an
+interactive job, where you launch your Matlab session. Which one you choose will depend on the workload you plan to have on the node before submitting a job.
+As for any other work on the cluster, you can always do minor work on the login node, but for heavier calculations (before submitting your multinode job), you should
+use an interactive job. 
+
+It is possible to use the Matlab GUI as well, on all login nodes. It is strongly recommended though to use NoMachine if you want to use the GUI though (login node
+3 and 4), as starting the GUI from any other login node will be extremely slow. If you want to use a GUI on a compute node, you'd need to submit your interacive job
+with X forwarding. You'll need to ``-X`` to your qsub command. Note that this is also quite slow. 
+
+Now you can start both interactive and independent batch jobs with the previously configured cluster profile. Follow the steps below to submit a job.
+
+Interactive job
+***************
+
+You can start an interactive job using the ``parpool`` function:
+
+::
+
+    >> c = parcluster;
+    >> p = parpool(64); % requesting 64 cores
+    
+Once the job has started, you'll receive output like this:
+
+::
+
+    Starting parallel pool (parpool) using the 'genius R2022a' profile ...
+
+    additionalSubmitArgs =
+
+        '-l nodes=2:ppn=32 -l pmem=4gb -A '<account_name>' -l walltime=00:30:00'
+
+    Connected to the parallel pool (number of workers: 64).
+
+    ans =
+
+        ClusterPool with properties:
+
+                Connected: true
+               NumWorkers: 64
```

Comment:
Using only 32 cores per node is something we don't generally recommend on Genius, so I think it's better to change the example to use 36 cores per node:
```suggestion
    >> p = parpool(72); % requesting 72 cores
    
Once the job has started, you'll receive output like this:

::

    Starting parallel pool (parpool) using the 'genius R2022a' profile ...

    additionalSubmitArgs =

        '-l nodes=2:ppn=36 -l pmem=4gb -A '<account_name>' -l walltime=00:30:00'

    Connected to the parallel pool (number of workers: 72).

    ans =

        ClusterPool with properties:

                Connected: true
               NumWorkers: 72
```